format/leveldb/leveldb_descriptor.go

```diff
@@ -0,0 +1,121 @@
+package leveldb
+
+// https://github.com/google/leveldb/blob/main/doc/impl.md#manifest
+// https://github.com/google/leveldb/blob/main/db/version_edit.cc
+//
+// Files in LevelDB using this format include:
+//  - MANIFEST-*
+
+import (
+	"embed"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed leveldb_log.md
+var leveldbDescriptorFS embed.FS
+
+func init() {
+	interp.RegisterFormat(
+		format.LevelDB_Descriptor,
+		&decode.Format{
+			Description: "LevelDB Descriptor",
+			DecodeFn:    ldbDescriptorDecode,
+		})
+	interp.RegisterFS(leveldbDescriptorFS)
+}
+
+const (
+	tagTypeComparator     = 1
+	tagTypeLogNumber      = 2
+	tagTypeNextFileNumber = 3
+	tagTypeLastSequence   = 4
+	tagTypeCompactPointer = 5
+	tagTypeDeletedFile    = 6
+	tagTypeNewFile        = 7
+	// 8 not used anymore
+	tagTypePrevLogNumber = 9
+)
+
+var tagTypes = scalar.UintMapSymStr{
+	tagTypeComparator:     "comparator",
+	tagTypeLogNumber:      "log_number",
+	tagTypeNextFileNumber: "next file number",
```

Comment:
Use underscore for these? i usually try use jq-ish constant unless the specification is very explicit about the naming